test/test_geo.py

```diff
@@ -144,15 +144,15 @@ def setUp(self):
 
     def test_bounding_box(self):
         bb = self.p.bounding_box
-        sw, ne = bb.corners
+        nw, se = bb.corners
```

Comment:
What is the reason for changing this?

Reply:
Documentation for `Rectangle` stated:
```
        """Create rectangle defined by its corners.

        :param .Point point_a: Top left corner.
        :param .Point point_b: Bottom right corner.
        """
```
But the location of `point_a` and `point_b` was not enforced anyway. And the behaviors of `Rectangle` class was fully dependent of passed arguments.

I add fix for that:
```python
        if point_a.latitude < point_b.latitude:
            point_a.latitude, point_b.latitude = point_b.latitude, point_a.latitude
        if point_a.longitude > point_b.longitude:
            point_a.longitude, point_b.longitude = point_b.longitude, point_a.longitude
```
And now `Rectangle` should be the same regardless of passed corners.

This enforcement changed the test a little.

Reply:
Makes sense.